eyetrackvr_backend/main.py

```diff
@@ -4,9 +4,9 @@
     os.chdir(os.path.dirname(__file__))
 
 from fastapi.staticfiles import StaticFiles
-from app.logger import setup_logger
+from .app.logger import setup_logger
 from fastapi import FastAPI
-from app.etvr import ETVR
+from .app.etvr import ETVR

```

Comment:
making this import relative seems to break things and prevent startup, reverting it to the state before resolves

Reply:
Hmm, how are you starting the application? I am running `python -m eyetrackvr_backend.main` which will treat the folder main resides in as a package

Reply:
i am running it directly with `poetry run python eyetrackvr_backend/main.py` but this relative package also cannot be found after being packaged with pyinstaller

Reply:
I see. Let me see how we can bend pyinstaller to run it as a module instead (i.e. equivalent to `poetry run python -m eyetrackvr_backend.main`)

Reply:
Seems like pyinstaller does not support modules properly? A workaround is to have a script outside the module directory that calls main.